tests/optimize/test_backtesting.py

```diff
@@ -645,8 +648,9 @@ def test_backtest_alternate_buy_sell(default_conf, fee, mocker, testdatadir):
     mocker.patch('freqtrade.optimize.backtesting.file_dump_json', MagicMock())
     backtest_conf = _make_backtest_conf(mocker, conf=default_conf,
                                         pair='UNITTEST/BTC', datadir=testdatadir)
-    # We need to enable sell-signal - otherwise it sells on ROI!!
-    default_conf['experimental'] = {"use_sell_signal": True}
+    # use_sell_signal is True by default now, it will sell on the sell signal instead of ROI,
+    # which is what we need here
+#    default_conf['ask_strategy'] = {"use_sell_signal": True}
```

Comment:
i guess since this is not needed anymore we can also remove this part (?) from the tests? (the comment does not make sense in the future - if you look at it from a "after the PR is merged" perspective ... (same with the others in this file ...)

Reply:
I thought I would be better 'from a "after the PR is merged" perspective' to emphasize that use_sell_signal: True is required for these tests... Okay, removed, even better...

Reply:
it's the new default - someone new will not know that this was different in the past, so might be confused by this comment